docs/resources/vpc_peering.md

```diff
@@ -41,12 +41,24 @@ Property | Description | Default
 `peer_region`| The Region code for the accepter VPC, if the accepter VPC is located in a Region other than the Region in which you make the request. | Not Required
 `aws_vpc_connection_id`| The ID of the VPC peering connection. | Computed
 
-#### Example
+#### AWS Example
 ```
-resource "instaclustr_vpc_peering" "example_vpc_peering" {
+resource "_instaclustr_vpc_peering" "example_vpc_peering" {
```

Comment:
Should this start with an underscore? Shouldn't it be `"instaclustr_vpc_peering"`?